docs/boiler/index.html

```diff
@@ -84,13 +84,13 @@ <h2 id="what-is-it%3F" tabindex="-1"><a class="bookmark" href="#what-is-it%3F"><
 <p>On top of that, it makes collaboration between engineering and design teams much easier and clearer. With B01LER, your teams can work smarter and create better products in less time.</p>
 <div class="button-group">
   <a class="cta-button" href="https://boilerds.com/">Visit Page <span aria-hidden="true">→</span></a>
-  <a class="cta-button" href="https://github.com/deven-org/B01LER-Kitchen" target="_blank">View on GitHub <span aria-hidden="true">→</span></a>
+  <a class="cta-button" href="https://github.com/deven-org/boiler" target="_blank">View on GitHub <span aria-hidden="true">→</span></a>
   </div>
   </div>
 
    <div class="project-info richtext"><h2 id="introduction" tabindex="-1"><a class="bookmark" href="#introduction"><span>Introduction</span></a></h2>
 <p>B01LER is a fully customisable design system (DS) that enables you to easily create cohesive and consistent digital products across multiple platforms and brands.</p>
-<p>B01LER provides pre-built components that exist <strong>as designs</strong> in the form of a <a href="https://boilerds.com/B01LER%20%20%5Bv0.1%20-%20alpha%20release%5D.fig" target="_blank" rel="noopener">component library in Figma</a>, and <strong>as code</strong> in the form of a <a href="https://github.com/deven-org/B01LER-Kitchen" target="_blank" rel="noopener">forkable repository</a>.</p>
+<p>B01LER provides pre-built components that exist <strong>as designs</strong> in the form of a <a href="https://boilerds.com/B01LER%20%20%5Bv0.1%20-%20alpha%20release%5D.fig" target="_blank" rel="noopener">component library in Figma</a>, and <strong>as code</strong> in the form of a <a href="https://github.com/deven-org/boiler" target="_blank" rel="noopener">forkable repository</a>.</p>
```

Comment:
The Figma link also was updated: https://www.figma.com/file/kG44mSWhEp2VcMvMqbJrRY/B01LER-%5Bv0.2--alpha-release%5D?type=design&node-id=1134%3A44297&mode=design&t=D8whh5YvFTH1immB-1